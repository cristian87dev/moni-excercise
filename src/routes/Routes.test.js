import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom'; // To prevent Link dependency error.
import renderer from 'react-test-renderer';
import Routes from './index';

it('Nav renders without crashing', () => {
  const component = renderer.create(
    <Router>
      <Routes />
    </Router>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
