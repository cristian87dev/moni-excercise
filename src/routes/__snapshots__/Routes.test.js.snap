// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders without crashing 1`] = `
<main
  className="main"
>
  <h1>
    Dashboard
  </h1>
</main>
`;
